### PR TITLE
saving data in databse

### PR DESCRIPTION
**Pull request by rafalkanski**
@filipgolonka

### CODE REVIEW

File: src/SaHackathon/Home/Api/Entity/Article.php [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+                    'title' => '?',
+                    'decision' => '?',
+                    'time' => '?',
+                    'user' => '?'
+                ]
+            )
+            ->setParameter(0, $data['title'])
+            ->setParameter(1, $data['decision'])
+            ->setParameter(2, $data['time'])
+            ->setParameter(3, $data['user'])
```
Comment (filipgolonka) [inline src/SaHackathon/Home/Api/Entity/Article.php]: article hash - ```hash```

**Comment by rafalkanski** [comment]
now @filipgolonka ?

**Comment by rafalkanski** [comment]
@filipgolonka what about that ?

**Comment by filipgolonka** [comment]
:+1:

**Pull request merged**